lib/mix/lib/mix/scm/git.ex

```diff
@@ -192,6 +192,15 @@ defmodule Mix.SCM.Git do
     end
   end
 
+  defp git_repos_match?(repo_a, repo_b) do
+    normalize_github_repo(repo_a) == normalize_github_repo(repo_b)
+  end
+
+  defp normalize_github_repo(repo) do
+    normalized = String.replace(repo, "git://git@github.com:", "https://github.com/")
```

Comment:
cool, I suspected there might be more idiomatic way of doing that. Nice, I like the syntax.

private git repos: right, removed